raiutils/raiutils/models/model_utils.py

```diff
@@ -60,6 +62,19 @@ def is_forecaster(model):
             model.forecast is not None)
 
 
+def is_object_detection_model(model):
+    """Check if the model is an object detection model.
+
+    :return: True if the model is a an object detection model,
+    False otherwise.
+    :rtype: bool
+    """
+    return (model is not None and
```

Comment:
Couldn't you leverage some code from is_classifier() and just check for hasattr(model, SKLearn.PREDICT) additionally?